src/components/CustomTextField/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { TextField } from '@material-ui/core';
+
+const CustomTextField = props => {
```

Comment:
hm let's take some time to think about naming - don't think CustomTextField looks the nicest in code but then TextField might be confusing

Reply:
I honestly don't know what's a better name... MuiTextField maybe? Not sure : (

Reply:
Ah naming - the hardest part of software engineering

Reply:
Okay how about InputField?

Reply:
yeah man idk - I think it should be textfield but I guarantee it'll be confusing for people

Reply:
idt MuiTextField is good cuz this is *our* custom TextField

Reply:
Yepppp 😩😩

Reply:
Let's finalize something tonight and stick with it for a bit